inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/ChartPanel.java

```diff
@@ -97,7 +95,7 @@ public void renderHead(IHeaderResponse aResponse)
                 "   $.ajax({",
                 "       url:'" + chartAjaxBejavior.getCallbackUrl().toString() + "',",
                 "       type:'post',",
-                "       cache:!1,",
+                "       cache:1,",
```

Comment:
According to the documentation http://api.jquery.com/jquery.ajax/ , we don't need the `cache` parameter in POST requests. For other requests, in my opinion, it would have made sense to set cache to false because the chart data changes between requests.